src/components/Faq.tsx

```diff
@@ -18,6 +18,7 @@ export default function Faq(): JSX.Element {
 					setExpanded={() => (index === expanded ? setExpanded(undefined) : setExpanded(index))}
 				></Expandable>
 			))}
+			<VersionNumber>v{import.meta.env.SNOWPACK_PUBLIC_VERSION}</VersionNumber>
```

Comment:
By letting `VersionNumber` be inside FAQ, it doesn't seem aligned to all sections. If you instead move it to `App.tsx`, the component renders just fine. See this example:

```
return (
		<AppContainer>
			<AppHeader />
			<AppContent>
				<Calculator></Calculator>
				<Faq></Faq>
			</AppContent>
			<VersionNumber>v{import.meta.env.SNOWPACK_PUBLIC_VERSION}</VersionNumber>
		</AppContainer>
	);
```

Reply:
That way, it doesn't matter if you open any FAQ expandable, or whatever the side of the screen is: The component will always be below all the app content

Reply:
Updated in [afd286e](https://github.com/ardriveapp/price-calculator/pull/28/commits/afd286e776dec5a7f321700d46b5a872bf16b698) 😄